dictionaries/condensed-strokes.json

```diff
@@ -1,4 +1,5 @@
 {
+"THAOE/AT/REUBG/KAL": "theatrical",
```

Comment:
@paulfioravanti this doesn't need to be added here since it is in `dict.json`

Reply:
Yep, you're right. Can't remember how I missed that. Will remove.